fw-child/clone-taxs.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
I would put this file in a new `scripts/` directory (can be under `fw-child/`). One day, we will reorganize the directories, and it will be easier if we can distinguish that this file is a script to run manually vs. a file used during template rendering.

Reply:
This has been completed. However, this is just a temporary script that will be used for content migration and will be deleted before launch.